### PR TITLE
added documentation for mentor api endpoints

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@hodgestar ready for review

**Comment by KaitCrawford** [comment]
I feel like it would be better to have the two API's on different pages but I'm not sure what structure would be best for that

**Comment by KaitCrawford** [comment]
The docs here are displaying these changes http://mentorship-tool.readthedocs.org/en/latest/index.html

File: docs/api_ref.rst [hunk 1]
```diff
@@ -177,4 +177,480 @@ Feedback
...
+    :param int sender_id:
+        The if of user who is sending the message.
+    :param int recipient_id:
+        The if of user who should receive the message.
+    :param str content: The content of the message"
+
+.. http:get:: /mentor/message/(message_id:str)
+
+    Return the message details if the message was sent or received by the
+    authenticated user. Otherwise, return 404
```
Comment (hodgestar) [inline docs/api_ref.rst]: Missing full stop. :)

**Comment by KaitCrawford** [comment]
@hodgestar ready for re-review

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**